libs/wingc/src/jsify/snapshots/base_class_with_fields_preflight.snap

```diff
@@ -98,7 +99,7 @@ class $Root extends $stdlib.std.Resource {
     }
     class Derived extends Base {
       constructor($scope, $id, ) {
-        super($scope, $id);
+        super($scope, $id, );
```

Comment:
Can this be avoided? Not critical if it's a hassle

Reply:
I was wondering about his change, it was generated after I added `super()` to the inflight constructor and I ran e2e tests. I don't know if this is critical, but seems suspicious to me, so I pushed it 🙂
<img width="347" alt="Screenshot 2023-12-28 at 11 16 45" src="https://github.com/winglang/wing/assets/16321871/fd255ed8-8191-4a69-a004-fc88dad86c41">